applications/initial_conditions/initial_conditions.cc

```diff
@@ -71,7 +71,10 @@ InitialConditionsNavierStokes<dim>::runTest()
                        this->present_solution,
                        this->exact_solution,
                        this->simulation_parameters.fem_parameters,
```

Comment:
Do you need to pass fem_parameters anymore? it seems like it is not used anymore because you pass the mapping, quadrature and etc.

Reply:
I removed it, good idea